inspect/README.md

```diff
@@ -0,0 +1,195 @@
+# forta-inspector-bot
+
+Inspects run environment capabilities and limits
+
+Moved from [forta-network/forta-inspector-bot](forta-inspector-bot-repo)
+
+# Inspection Details
+
+## Scan API Inspection
+
+### `scan-api.accessible`
+
+Checks Scan APIs accessibility by establishing rpc connection and asking for current block height
+
+`1` for accessible, `-1` for not
+
+### `scan-api.module.eth`
+
+The scan node should either declare `eth` module in `rpc_modules` method or it should return a valid response to
+a `eth_chainId` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.module.net`
+
+The scan api should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.block-by-number.hash`
+
+The hash of the block data retrieved from the scan API
+
+## Trace API Inspection
+
+### `trace-api.accessible`
+
+Checks trace APIs accessibility by establishing rpc connection and asking for current its chain id.
+
+`1` for accessible, `-1` for not
+
+### `trace-api.supported`
+
+Only necessary for Ethereum Mainnet.
```

Comment:
It's also necessary for Fantom. Let's remove this line entirely and not assume what's it available for. We can keep https://docs.forta.network/en/latest/scanner-quickstart/ as the source of truth to this.

Reply:
Fantom trace inspection could need some inspection code changes as well, i think have a mainnet only condition somehere

Reply:
Yeah, great point. I had the same concern, it will become adjustable per-chain with this PR: https://github.com/forta-network/forta-node/pull/539
The inspection config uses that same "need checking" value in `forta-node`.